bears/js/JSONFormatBear.py

```diff
@@ -1,23 +1,16 @@
 import json
 from collections import OrderedDict
 
+from coala_utils.param_convertion import negate
```

Comment:
`coala_utils.param_convertion` should be renamed to `coala_utils.param_conversion`, see https://gitlab.com/coala/coala-utils/merge_requests/22